src/extension.ts

```diff
@@ -10,6 +10,15 @@ import * as path from 'path';
 import * as Url from 'url';
 import * as fs from 'fs';
 
+function checkValidEditor() : boolean {
+    if (vscode.window.activeTextEditor === undefined) {
```

Comment:
For my own clarification (on phone at the moment, so can't try myself), if the document is too large, exactly what does VSCode do?  It must be giving a window (since users are activating the method), but trying to access it gives back an undefined reference?